packages/opds-web-client/src/components/BookDetails.tsx

```diff
@@ -61,9 +60,6 @@ export default class BookDetails<P extends BookDetailsProps> extends Book<P> {
                 { this.circulationInfo() }
               </div>
             </div>
-            <div className="right-column-links col-sm-3">
-              { this.rightColumnLinks() }
-            </div>
```

Comment:
Why did you take this out? The patron app overrides `rightColumnLinks` to put in the report a problem link.

Reply:
Maybe it needs a comment that it's intentionally empty.

Reply:
Ah yes, that would be better.